Docs/deep_linking.md

```diff
@@ -0,0 +1,78 @@
+# URL structure for deeplinking
+
+ This describes the structure of the URL so that it is easy for developers to create deep links from other applications directly to a well/wellbore/object/curve data etc.
+
+## Common parameters:
+### objectGroup
+
+This paramater can have values:
+  - BhaRun
+  - ChangeLog
+  - FluidsReport
+  - FormationMarker
+  - Message
+  - MudLog
+  - Log
+  - Rig
+  - Risk
+  - Trajectory
+  - Tubular
+  - WbGeometry
+
+### logType
+
+This paramater can have values:
+  - time
+  - depth
+
+### filterType
+
+This paramater can have values:
+  - Log
+  - Group
+  - ServiceCompany
```

Comment:
Seems incorrect. The values can be Wellbore, Log, Rig or Service%20Company.

Reply:
done